.github/workflows/docker-edge.yml

```
@@ -37,6 +37,10 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu, alpine]
+        platform: [linux/amd64, linux/arm64, linux/arm/v7, linux/arm/v6]
```

Suggestion: removing the `linux/` prefix makes it possible to read the whole arch name in the sidebar of the action run page (eg https://github.com/intiplink/actual-server/actions/runs/4638998275/jobs/8209365335)

```suggestion
        arch: [amd64, arm64, arm/v7, arm/v6]
```

Then you can say  `platform: linux/${{ matrix.arch }}` below. What do you think?

---

make sense, lemme edit it